httpclient/pom.xml

```diff
@@ -55,6 +55,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
I don't think this is being used, is it?

Reply:
it's required for test as `jsr311-api` is not pulled in transitively any longer. will add `scope`.